src/main/kotlin/org/jitsi/jicofo/util/QueueExecutor.kt

```diff
@@ -35,7 +35,7 @@ class QueueExecutor(
 ) : Executor, PacketQueue<Runnable>(
     Integer.MAX_VALUE,
     true,
-    id,
+    "QueueExecutor",
```

Comment:
I added the ID field in case `JitsiMeetConferenceImpl` is not the only user of `QueueExecutor`. If this is unnecessary, we should also remove the `id` param from the constructor.

Reply:
Ah, it's got a static name, rather than a per-conference name.  In that case the passed-in `id` is fine.